components/docs-chef-io/content/automate/monitoring.md

```diff
@@ -60,7 +60,7 @@ To use `/status`, set up an authentication token for use with your monitoring sy
 3. Test that your token and policy give you access to the `/status` endpoint by running the following command:
 
     ```bash
-    curl -k -H "api-token: <token-id>" https://automate.example.com/api/v0/status?pretty
+    curl -k -H "api-token: <monitoring-token>" https://automate.example.com/api/v0/status?pretty
```

Comment:
lets make this change here as well to avoid any confusion 
in L48
1. Generate a token:
```
1. Generate a monitoring token:
```
in L54
2. Create a policy that allows your created token to access the `/status` endpoint.
```
2. Create a policy that allows your created monitoring token to access the `/status` endpoint.
```